command.py

```diff
@@ -108,3 +108,21 @@ class ExitCommand(Command):
     def execute(self):
         # TODO: Find more elegant way to terminate the program
         exit()
+
+
+class PickupCommand(Command):
+
+    """
+    Pickup an item at the player's current position.
+    """
+
+    def __init__(self, player, game_map):
+        self.player = player
+        self.game_map = game_map
+
+    def execute(self):
+        loot = [e for e in self.game_map.entities if e.pos == self.player.pos and e != self.player]
```

Comment:
this is kinda ugly...

Reply:
If we only want the players to be able to grab items on the same tile as them, a possible solution could be to have a list of references to objects that the player is "stepping on" each turn, like some other games do. So when you move to a tile with items in it, you'd get a message saying "There is an X here." or something like that. Then, by pressing the grab key, you'd get a list of the objects that are on the tile and you'd select the one or ones you want to grab. Some games that do this are for example Nethack, Cataclysm: DDA or Unreal World (although this last one also allows you to grab items on adjacent tiles with another key, not sure about the others now).

Reply:
We can do that indeed

Reply:
Wait no, to do that we'd have to do the same (look for entities with the same pos as the player) but we'd have to do it every turn even if the player doesn't want to pick them up...

Reply:
Still a nice feature to consider.